### PR TITLE
Fix broken link

### LINKED ISSUE #8635: [v.7.0] /docs/pages/application-access/controls.mdx
## Details
The Link for Gsuite integration is broken on this page https://goteleport.com/docs/application-access/controls/
This links to azured setup and not the Gsuite setup

image

The correct link is to `../enterprise/sso/google-workspace.mdx`

This is mistake is present in master, v8, v7, and v6.2, though the 6.2 url is slightly different: `../enterprise/sso/ssh-google-workspace.mdx`

### Category

- Improve Existing

### PR DESCRIPTION
**Pull request by Erick-Reyes**
Contributes to: https://github.com/gravitational/teleport/issues/8635

### CODE REVIEW

**Review by wadells — APPROVED** [review]

**Review by ptgott — APPROVED** [review]

File: docs/pages/application-access/controls.mdx [hunk 1]
```diff
@@ -79,6 +79,6 @@ allow:
   - [OIDC](../enterprise/sso/oidc.mdx)
   - [ADFS](../enterprise/sso/adfs.mdx)
   - [Azure AD](../enterprise/sso/azuread.mdx)
-  - [GSuite](../enterprise/sso/azuread.mdx)
+  - [GSuite](../enterprise/sso/google-workspace.mdx)
```
Comment (russjones) [inline docs/pages/application-access/controls.mdx:82]: GSuite -> Google Workspace

Comment (russjones) [reply docs/pages/application-access/controls.mdx:82]: Then please also update:

* https://github.com/gravitational/teleport/pull/9831
* https://github.com/gravitational/teleport/pull/9829
* https://github.com/gravitational/teleport/pull/9828

Comment (Erick-Reyes) [reply docs/pages/application-access/controls.mdx]: Fix this for all versions, probably they need to be review as well. Thank you.

**Review by zmb3 — APPROVED** [review]

**Comment by russjones** [comment]
Will buddy merge in #10777.

**Pull request closed without merging**